preprocessing/dummy/lineage-alternative.yaml

```
@@ -0,0 +1,33 @@
+A:
+  aliases: []
+  parents: []
+A.1:
+  aliases: []
+  parents:
+  - A
+A.2:
+  aliases: []
+  parents:
+  - A
+A.1.1:
+  aliases: []
+  parents:
+  - A.1
+  - A.2
+B:
+  aliases: []
+  parents: []
+B.1:
+  aliases: []
+  parents:
+  - B
+B.1.1:
+  aliases:
+  - C
```

```suggestion
C:
  aliases:
  - B.1.1
```
I think from previously this is the intended set up?

---

nono, it was deliberately done the way it is. @chaoran-chen told me that it is supposed to look like that.

---

you only use aliases once you have a number. So you wouldn't use "C" as a lineage def on its own. That was my understanding.